<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Welcome to Digital Signer App!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>What do you want to do?</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="digiSignButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Digital Sign</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>741</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify Digital Sign</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
